ui/src/style/variables.less

```
@@ -17,6 +17,9 @@
 @equation-user-annotation-color: #e82;
 @symbol-user-annotation-color: #f5f;
 
+@button-background-color: #1857b6;
```

nit pick: perhaps this naming to make it clearer which buttons this style will apply to?

```suggestion
@s2-primary-action-background-color: #1857b6;
```